docs/docs/sources/whatsapp-cloud.md

```diff
@@ -0,0 +1,128 @@
+---
+title: WhatsApp
+sidebar_label: WhatsApp
+---
+
+import TLDR from "@site/src/components/TLDR";
+import SuccessBox from "@site/src/components/SuccessBox";
+import ButtonBox from "@site/src/components/ButtonBox";
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::warning
+
+This source is still under development. You can track the progress [here](https://github.com/airyhq/airy/milestone/38).
```

Comment:
Tyring a new process whereby we explicitly document the state of a component and in return can do more work on the docs in each PR